dask/array/tests/test_fft.py

```diff
@@ -195,3 +195,9 @@ def test_wrap_fftns(modname, funcname, dtype):
         wfunc(darr2c, (darr2c.shape[0] - 1, darr2c.shape[1] - 1), (0, 1)),
         func(nparrc, (nparrc.shape[0] - 1, nparrc.shape[1] - 1), (0, 1))
     )
+
+
+@pytest.mark.parametrize("n", list(range(1, 10)))
+@pytest.mark.parametrize("d", [1.0, 0.5, 2 * np.pi])
```

Comment:
Maybe not, was just trying to be thorough. Certainly would want to check an even, and an odd at least. FWIW this takes ~0.1s to run as it is.

As for `parametrize`, this is something that @mrocklin has encouraged me to do in the past. Am kind of ok with it as it makes it easier to tell what combination failed. That said, am not really picky about this.